README.rst

```diff
@@ -60,7 +60,8 @@ Environment variables available to run and test the job:
     Representative packages for Computer Vision (CV).
 - **THOTH_PRESCRIPTIONS_DRY_RUN**
     If enabled, prescriptions won't be created but only printed in logs.
-
+- **THOTH_PRESCRIPTIONS_REFRESH_SCHEME_PROTOCOL**
```

Comment:
Might be a good idea to specify that it is for user-api. There are done HTTPS calls to other services as well.

Reply:
👍🏼 sounds good, thanks @fridex !